src/GraphQL/Execution/ExecutionResult.cs

```diff
@@ -76,6 +77,19 @@ public ExecutionResult(ExecutionResult result)
             Extensions = result.Extensions;
         }
 
+        /// <summary>
+        /// Initializes a new instance with the <see cref="Query"/>, <see cref="Document"/>,
+        /// <see cref="Operation"/> and <see cref="Extensions"/> properties set from the
+        /// specified <see cref="ExecutionContext"/>.
+        /// </summary>
+        internal ExecutionResult(ExecutionContext context)
```

Comment:
Looks like this code can be placed directly at caller side. All properties are public settable. I would not bother with additionala internal ctors here.

Reply:
It is used three or four different times in the code and I think one of the properties was missed in one place (output extensions?).  I didn't want to miss setting a property in the future so I consolidated the code.  It's internal and should not negatively affect the codebase.  Ok?

Reply:
Well, OK, although such selective copying looks strange.

Reply:
Perhaps I can improve the xml comment.

Reply:
I don't have a good answer here either.  Leaving as-is.